gradle/wrapper/gradle-wrapper.properties

```
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
```

We use gradle v7.5.1 for `corda-payments-sdk`, so for the consistency, please, update the version
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
```